src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return dayPrice * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (dayPrice * days) - smallDiscount;
+  }
+
+  if (days >= 7) {
+    return (dayPrice * days) - bigDiscount;
+  }
```

Comment:
pass days count needed for discount into variables too

Reply:
+